spec/acceptance/openldap__server__dbindex_spec.rb

```diff
@@ -15,6 +15,7 @@ class { 'openldap::server':
         }
         Openldap::Server::Dbindex {
           suffix => 'dc=foo,dc=example,dc=com',
+          require => Openldap::Server::Database['dc=foo,dc=example,dc=com'],
```

Comment:
If this is required then maybe we need some kind of automatic dependency somewhere